### PR TITLE
Merge tag 'v4.8.0' into v5.x.x

### PR DESCRIPTION
**Pull request by ben-palmer-sociomantic**

### CODE REVIEW

File: src/ocean/text/json/Json.d [hunk 1]
```diff
@@ -31,60 +31,62 @@ import ocean.text.json.JsonParser;
 
 import Float = ocean.text.convert.Float;
 
-version (unittest) import ocean.core.Test;
+version (UnitTest)
```
Comment (Geod24) [inline src/ocean/text/json/Json.d]: Wrong

**Review by Geod24 — CHANGES_REQUESTED** [review]

File: src/ocean/text/json/Json.d (hunk 1 above)
Comment (ben-palmer-sociomantic) [reply src/ocean/text/json/Json.d]: Should it stay as `version (unittest)`?

**Comment by ben-palmer-sociomantic** [comment]
Marking as blocked until I have done the required bug fix to v4.8.x then this should be a simple merge.

File: src/ocean/text/json/Json.d (hunk 1 above)
Comment (Geod24) [reply src/ocean/text/json/Json.d]: Yes

**Comment by ben-palmer-sociomantic** [comment]
Closing without merging as not needed.

**Pull request closed without merging**